docs/validators/staking-v4.md

```diff
@@ -0,0 +1,338 @@
+---
+id: staking-v4
+title: Staking v4
+---
+
+# **Introduction**
+
+Staking and delegation are processes that evolve over time. No system has to remain static. Our assumptions about how
+the market works and reacts can change, just as user behavior and market dynamics may evolve. Currently, we have
+approximately 400 validators, with some acting as staking providers and others as individual validator operators. While
+most nodes have a comfortable top-up on the base stake of 2.5k eGLD, some do not contribute to the network's security by
+adding more top-up.
+
+# **Problems with the Current Implementation**
+
+The issues with the current implementation include:
+
+- Limiting the number of nodes to 3200, creating an additional queue. New validators can join the network only if
+  someone leaves the system.
+- Concentration of power among large providers, hindering decentralization. MultiversX Foundation together with the top
```

Comment:
I would rephrase it, say that the top X agencies control 33%.

Reply:
Right, good point, rephrased it